aspnet/TreeView/Template-Option.md

```diff
@@ -87,7 +87,7 @@ In the view page, specify template format and add TreeView element
             {{"{{"}}else{{}}}}
                 <div class="cont-list">
                     <img class="con-img"
```

Comment:
Don't update the http url in code section. We need to get product team confirmation, then only we can change here.